services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js

```diff
@@ -14,32 +14,7 @@ const columns = [
   { title: "Last Amended", dataIndex: "last_amended", key: "last_amended", sorter: true },
 ];
 
-const data = [
-  {
-    key: "MX-ABC",
-    permit_no: "MX-ABC",
-    permit_status: "Open",
-    permit_end_date: "December 31, 2020",
-    first_issued: "April 16, 2011",
-    last_amended: "February 13, 2019",
-  },
-  {
-    key: "MX-DEF",
-    permit_no: "MX-DEF",
-    permit_status: "Closed",
-    permit_end_date: "December 31, 2018",
-    first_issued: "April 12, 2008",
-    last_amended: "July 8, 2017",
-  },
-  {
-    key: "MX-GHI",
-    permit_no: "MX-GHI",
-    permit_status: "Open",
-    permit_end_date: "August 7, 2021",
-    first_issued: "April 12, 2015",
-    last_amended: "February 1, 2017",
-  },
-];
+const data = [];
```

Comment:
Is there a reason for this empty list?

Reply:
It's just placeholder for whoever will be working on the permits table. :)